tests/cupy_tests/linalg_tests/test_decomposition.py

```diff
@@ -15,14 +15,26 @@ class TestCholeskyDecomposition(unittest.TestCase):
 
     @testing.for_float_dtypes(no_float16=True)
```

Comment:
I manually implemented this feature, but found that float16 is too insufficient to preserve positive definite. Since the numpy does not contain this feature, I do not recommend to explicitly add such additional feature.